src/v2/api/group/GroupService.ts

```diff
@@ -69,4 +54,26 @@ export class GroupService {
 
     return results;
   }
+
+  async getCaptain(groupId: string) {
+    const students = await this.groupRepository.getStudents(groupId);
+    for (const student of students) {
+      const roles = await this.studentRepository.getRoles(student.userId);
```

Comment:
it's not an array of roles names, it's an array of role entity. name is stored in the field of role entity.